src/web/ng/Server.hpp

```diff
@@ -42,6 +44,19 @@ namespace web::ng {
  * @brief Web server class.
  */
 class Server {
+public:
+    /**
+     * @brief Check to perform for each new client connection. The check takes client ip as input and returns a Response
```

Comment:
It can be : Check to perform for each new client connection. The check takes client ip as input and returns a Response _if check fails_ .